sjb/actions/oct_install.py

```diff
@@ -27,8 +31,13 @@ class OCTInstallAction(Action):
     the tool should `source activate` the venv.
     """
 
-    def generate_build_steps(self):
+    def generate_build_steps(self, output_format):
+        oct_template = _OCT_INSTALL_ACTION_FULL
+        if output_format == "sh":
```

Comment:
Or we can set up the Dockerfile to have the same `sjb` installation pattern?

Reply:
Done, we're faking venv now